package.json

```diff
@@ -115,6 +114,7 @@
     "identity-obj-proxy": "3.0.0",
     "jest-canvas-mock": "^2.4.0",
     "jest-localstorage-mock": "^2.4.22",
+    "lodash": "^4.17.21",
```

Comment:
[inform] See other related comments, but I believe we can fully remove this `lodash` module from the `package.json`, given it seems the tests do pass without needing its usage in `__mocks__`.